src/inc/ContentType.hpp

```diff
@@ -0,0 +1,22 @@
+//
+//  Copyright (C) 2019 Microsoft.  All rights reserved.
+//  See LICENSE file in the project root for full license information.
+// 
+//  Helper file for determining content type and compression settings for adding files to a package
+#pragma once
+
+#include "AppxPackaging.hpp"
+
+#include <map>
+
+namespace MSIX {
+
+    typedef std::pair<std::string, APPX_COMPRESSION_OPTION> ContentTypeData;
+
+    class ContentType
+    {
+    public:
+        static const ContentTypeData GetExtensionContentTypeData(std::string& ext);

```

Comment:
>ContentTypeData [](start = 21, length = 15)

Just my own stylistic here: if you are going to have these be static, they should return a ContentType.

Reply:
Also, why not const ContentType&

---
In reply to: [278644277](https://github.com/Microsoft/msix-packaging/pull/95#discussion_r278644277) [](ancestors = 278644277)